app/views/warehouse_reports/health/eligibility_results/_table.haml

```diff
@@ -0,0 +1,18 @@
+%table.table.table-condensed
+  %thead
+    %tr
+      %th Name
+      %th DOB
+      %th ACO Coverage
+      %th Agency
+      %th Care Coordinator
+  %tbody
+    - patients.each do |patient|
+      %tr
+        %td= link_to "#{patient.first_name} #{patient.last_name}", window_client_health_patient_index_path(patient.client)
```

Comment:
There are some helper methods for these and polymorphic path.  We should use those.  See app/controllers/concerns/combined_client_paths_generator.rb